core/src/test/scala/unit/kafka/log/LogManagerTest.scala

```diff
@@ -949,6 +949,7 @@ class LogManagerTest {
     appendRecordsToLog(mockTime, logDir1, 0, mockBrokerTopicStats, expectedSegmentsPerLog)
     appendRecordsToLog(mockTime, logDir2, 1, mockBrokerTopicStats, expectedSegmentsPerLog)
 
+    val noOpsScheduler = new NoOpsScheduler()
```

Comment:
How about creating a (noop) `mock` scheduler? that is more simple :)

Reply:
Yes, using `mock` can have same effect. Thank you.